test/e2e/network_test.go

```diff
@@ -265,6 +265,18 @@ var _ = Describe("Podman network", func() {
 		Expect(rmAll.ExitCode()).To(BeZero())
 	})
 
+	It("podman network remove no-network", func() {
+		session := podmanTest.Podman([]string{"network", "rm", "no-network"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(1))
+	})
+
+	It("podman network remove ", func() {
+		session := podmanTest.Podman([]string{"network", "rm", "no-network"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(1))
```

Comment:
It might be time for a new cup of tea, but isn't the testing you're doing here the same as the previous test?

Reply:
Yes, this is a cut and paste. Will remove second.